adafruit_lis3dh.py

```diff
@@ -284,14 +306,14 @@ def tapped(self):
 
     def set_tap(
         self,
-        tap,
-        threshold,
+        tap: Literal[0, 1, 2],
+        threshold: int,
         *,
-        time_limit=10,
-        time_latency=20,
-        time_window=255,
-        click_cfg=None
-    ):
+        time_limit: Optional[int] = 10,
+        time_latency: Optional[int] = 20,
+        time_window: Optional[int] = 255,
```

Comment:
So `Optional` has a misleading name - it doesn't mean that an argument is optional to provide, but rather that it can take `None` as a value.  Therefore, `time_limit`, `time_latency`, and `time_window` should not be `Optional` since they do need to be `int` values, so they should just be that.

Reply:
Tks see https://github.com/adafruit/Adafruit_CircuitPython_LIS3DH/pull/76/commits/59c669238f330e5703bb89641199c02303db38c7